src/utils/api.js

```diff
@@ -0,0 +1,6 @@
+import { WATCHLIST } from 'constants/constants';
+
+export const setWatchlist = array => {
+  return localStorage.setItem(WATCHLIST, JSON.stringify(array));
+};
+export const getWatchlist = () => localStorage.getItem(WATCHLIST) ?? '[]';
```

Comment:
I was thinking that we can return the JSON already parsed
```suggestion
export const getWatchlist = () => JSON.parse(localStorage.getItem(WATCHLIST) || '[]');
```